repos/system_upgrade/el7toel8/actors/repositoriesblacklist/libraries/repositoriesblacklist.py

```diff
@@ -2,12 +2,69 @@
 from leapp.libraries.common.config import get_product_type
 from leapp.libraries.stdlib import api
 from leapp.models import (
+    CustomTargetRepository,
     RepositoriesBlacklisted,
     RepositoriesFacts,
     RepositoriesMap,
 )
 
 
+def _report_using_unsupported_repos(repos):
+    report = [
+        reporting.Title("Using repository not supported by Red Hat"),
+        reporting.Summary(
+            "The following repositories has been used for the "
+            "upgrade, but they are not supported by the Red Hat.:\n"
+            "- {}".format("\n - ".join(repos))
+        ),
+        reporting.ExternalLink(
```

Comment:
I am confused by this report, and I am pretty sure customers would be confused too.

You talk about unsupported repositories and add a link to CRB - And from the looks of the code it seems like that this link even would come up if none of the repos is the CRB repo. Or am I missing something?

Similar for the black listed repo.

Reply:
that was kind of the problem already before. Maybe we should think about the better wording.

Reply:
btw: `following repositories have`.

Reply:
Solved in https://github.com/oamg/leapp-repository/pull/623/commits/f9511d08273b1c6fcaf98190188b95c5fcb338b9